rita/Cargo.toml

```diff
@@ -25,8 +25,8 @@ exit_db = { path = "../exit_db" }
 num256 = { path = "../num256" }
 settings = { path = "../settings" }
 
-actix = "0.5.7"
-actix-web = { version = "0.6.7", default-features = false}
+actix = "0.5.8"
+actix-web = { version = "0.6.12", default-features = false}
```

Comment:
out of curiosity what did you upgrade for?

Reply:
It had some new features for the http client which made this possible